bin/compile

```diff
@@ -403,6 +412,8 @@ trap clearGitCredHelper INT TERM EXIT
 
 determineTool
 
+mcount "version.$ver"
```

Comment:
There are other binaries we might track versions of, like `dep`, etc. How about making this:

```
mcount "version.go.$ver
```

Reply:
isn't that handled by BPLOG_PREFIX?

Reply:
Oh nevermind. Now I understand. I'm slow. ;-)

Reply:
@freeformz @jmorrell these are excellent suggestions. What do you think of 
BPLOG_PREFIX="buildpack.golang" with mcount "version.go.$ver

Reply:
I lean towards `BPLOG_PREFIX="buildpack.go"` so that it matches the buildpack name, but either works.